dali/ft/filecopy.cpp

```diff
@@ -2917,6 +2920,7 @@ void FileSprayer::spray()
     }
 
 
+    //Get the size of the source files directly (to check they exist).
     gatherFileSizes(true);
```

Comment:
without having called getFileSizes(false) earlier, this will now fail if primary copies are missing but replicates exist.
See other comment where getFileSizes(false) call removed.

Reply:
Reverted